skymap_scanner/server/utils.py

```diff
@@ -8,24 +8,45 @@
 from typing import Any, Dict, List, Optional, Tuple
 
 import cachetools.func
-from rest_tools.client import RestClient
+from rest_tools.client import CalcRetryFromWaittimeMax, RestClient
 
 from .. import config as cfg
 from . import LOGGER
 
 
-def fetch_event_contents(
-    event_file: Optional[Path], skydriver_rc: Optional[RestClient]
-) -> Any:
-    """Fetch event contents from file (.json or .pkl) or via SkyDriver."""
-    # request from skydriver
-    if skydriver_rc:
-        manifest = skydriver_rc.request_seq(
-            "GET", f"/scan/{cfg.ENV.SKYSCAN_SKYDRIVER_SCAN_ID}/manifest"
+def connect_to_skydriver(urgent: bool) -> RestClient:
+    """Get REST client for SkyDriver depending on the urgency."""
+    if urgent:
+        return RestClient(
+            cfg.ENV.SKYSCAN_SKYDRIVER_ADDRESS,
+            token=cfg.ENV.SKYSCAN_SKYDRIVER_AUTH,
+            timeout=60.0,
+            retries=CalcRetryFromWaittimeMax(waittime_max=1 * 60 * 60),
+            # backoff_factor=0.3,
         )
-        LOGGER.info("Fetched event contents from SkyDriver")
-        return manifest["event_i3live_json_dict"]
+    else:
+        return RestClient(
+            cfg.ENV.SKYSCAN_SKYDRIVER_ADDRESS,
+            token=cfg.ENV.SKYSCAN_SKYDRIVER_AUTH,
+            timeout=10.0,
+            retries=0,
```

Comment:
Do we really want no retries at all?  This means it's OK for these requests to drop on the floor if there's a transient error.